core/templates/pages/topic-viewer-page/topic-viewer-page.module.ts

```diff
@@ -44,17 +44,24 @@ import { ObjectsDomainConstants } from
 import { TopicViewerDomainConstants } from
   'domain/topic_viewer/topic-viewer-domain.constants';
 
+import { TopicViewerNavbarBreadcrumbComponent } from
+  './navbar-breadcrumb/topic-viewer-navbar-breadcrumb.component';
+import { PracticeTabComponent } from './practice-tab/practice-tab.component';
+
+
 @NgModule({
   imports: [
     BrowserModule,
     HttpClientModule,
     SharedComponentsModule
   ],
   declarations: [
-    OppiaAngularRootComponent
+    OppiaAngularRootComponent, PracticeTabComponent,
```

Comment:
I am not entirely sure about this, but i think i have seen that we add one item per line. Do verify this, I am not sure.

Reply:
A comment on 9673 addressed this. Just going to sort it all later.